python/sglang/multimodal_gen/docs/cache_dit.md

```diff
@@ -0,0 +1,182 @@
+which to use cached results.
+
+#### SCM Presets
+
+SCM is configured with presets:
+
+| Preset   | Compute Ratio | Speed    | Quality    |
+|----------|---------------|----------|------------|
+| `none`   | 100%          | Baseline | Best       |
+| `slow`   | ~75%          | ~1.3x    | High       |
+| `medium` | ~50%          | ~2x      | Good       |
+| `fast`   | ~35%          | ~3x      | Acceptable |
+| `ultra`  | ~25%          | ~4x      | Lower      |
+
+##### Usage
+
+```bash
+SGLANG_CACHE_DIT_ENABLED=true \
+SGLANG_CACHE_DIT_SCM_PRESET=medium \
+sglang generate --model-path Qwen/Qwen-Image \
+    --prompt "A futuristic cityscape at sunset"
+```
+
+#### Custom SCM Bins
+
+For fine-grained control over which steps to compute vs cache:
+
+```bash
+SGLANG_CACHE_DIT_ENABLED=true \
+SGLANG_CACHE_DIT_SCM_COMPUTE_BINS="8,3,3,2,2" \
+SGLANG_CACHE_DIT_SCM_CACHE_BINS="1,2,2,2,3" \
+sglang generate --model-path Qwen/Qwen-Image \
+    --prompt "A futuristic cityscape at sunset"
+```
+
+#### SCM Policy
+
+| Policy    | Env Variable                          | Description                                 |
+|-----------|---------------------------------------|---------------------------------------------|
+| `dynamic` | `SGLANG_CACHE_DIT_SCM_POLICY=dynamic` | Adaptive caching based on content (default) |
+| `static`  | `SGLANG_CACHE_DIT_SCM_POLICY=static`  | Fixed caching pattern                       |
+
+## Environment Variables
+
+All Cache-DiT parameters can be set via the following environment variables:
+
+| Environment Variable                | Default   | Description                              |
+|-------------------------------------|-----------|------------------------------------------|
+| `SGLANG_CACHE_DIT_ENABLED`          | `false`   | Enable Cache-DiT acceleration            |
+| `SGLANG_CACHE_DIT_FN`               | `1`       | First N blocks to always compute         |
+| `SGLANG_CACHE_DIT_BN`               | `0`       | Last N blocks to always compute          |
+| `SGLANG_CACHE_DIT_WARMUP`           | `8`       | Warmup steps before caching              |
+| `SGLANG_CACHE_DIT_RDT`              | `0.35`    | Residual difference threshold            |
+| `SGLANG_CACHE_DIT_MC`               | `3`       | Max continuous cached steps              |
+| `SGLANG_CACHE_DIT_TAYLORSEER`       | `true`    | Enable TaylorSeer calibrator             |
+| `SGLANG_CACHE_DIT_TS_ORDER`         | `1`       | TaylorSeer order (1 or 2)                |
+| `SGLANG_CACHE_DIT_SCM_PRESET`       | `none`    | SCM preset (none/slow/medium/fast/ultra) |
+| `SGLANG_CACHE_DIT_SCM_POLICY`       | `dynamic` | SCM caching policy                       |
+| `SGLANG_CACHE_DIT_SCM_COMPUTE_BINS` | not set   | Custom SCM compute bins                  |
+| `SGLANG_CACHE_DIT_SCM_CACHE_BINS`   | not set   | Custom SCM cache bins                    |
+
+## Supported Models
+
+Cache-DiT supports DiT models:
```

Comment:
`Cache-DiT` -> `SGLang Diffusion x Cache-DiT` would be better ? In fact, cache-dit supports almost all DiT models.